application/admin/views.py

```diff
@@ -210,3 +210,15 @@ def data_sources():
     data_sources = DataSource.query.order_by(DataSource.title).all()
 
     return render_template("admin/data_sources.html", data_sources=data_sources)
+
+
+@admin_blueprint.route("/data-sources/merge")
+@login_required
+@user_can(MANAGE_USERS)  # TODO: update to MANAGE_DATA_SOURCES
+def merge_data_sources():
+
+    data_source_ids = request.args.get("ids").split(",")
```

Comment:
This will need to change to `request.args.getlist("data_sources")`.

A GET form with Design System checkboxes requests a URL like `http://localhost:5000/admin/data-sources?data_sources=864&data_sources=924`. There are multiple `data_sources` keys, so we need to get all of the values for that key at once, which `getlist` does (part of the [MultiDict](https://werkzeug.palletsprojects.com/en/0.15.x/datastructures/#werkzeug.datastructures.ImmutableMultiDict)).